mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -32,19 +33,32 @@ def cross_entropy(pred,
             If None, it will be set to default value. Default: -100.
         avg_non_ignore (bool): The flag decides to whether the loss is
             only averaged over non-ignored targets. Default: False.
-
+        label_smoothing (float, optional): A float in range [0.0, 1.0].
+            Specifies the amount of smoothing when computing the loss, where
+            0.0 means no smoothing. Defaults to 0.0.
     Returns:
         torch.Tensor: The calculated loss
     """
     # The default value of ignore_index is the same as F.cross_entropy
     ignore_index = -100 if ignore_index is None else ignore_index
     # element-wise losses
-    loss = F.cross_entropy(
-        pred,
-        label,
-        weight=class_weight,
-        reduction='none',
-        ignore_index=ignore_index)
+
+    if torch.__version__ < '1.10':
+        loss = F.cross_entropy(
```

Comment:
This comparison may be wrong. we can use the `digit_version` in `mmcv.utils.version_utils`